### ISSUE TITLE
is_utc() returns False when parsing from a UTC date

### ISSUE THREAD

**Issue by joaonc** [sdispater/pendulum#293]
This date is UTC: `'2018-06-21T09:30:00+00:00'`, however, the `is_utc()` method returns `False`.

```
>>> pendulum.parse('2018-06-21T09:30:00+00:00').timezone
Timezone('+00:00')

>>> pendulum.parse('2018-06-21T09:30:00+00:00').is_utc()
False
```

The current implementation is:
```
    def is_utc(self):
        return self.timezone_name == 'UTC'
```
Maybe it can be something like:
```
    def is_utc(self):
        return self.timezone.offset == 0
```

**Comment by redlickigrzegorz** [comment]
I created a pull request #295 for resolving this issue. I hope it resolves the problem. I would like to ask for a review also.

**Comment by joaonc** [comment]
@darrenleeweber Are you saying/asking something or simply demonstrating cases where it returns `False` when should be returning `True`?  (ie, reproing the bug)

**Comment by darrenleeweber** [comment]
Yes, reproducing the behavior in 1.5.1 vs 2.x is important for me (and possibly others landing here).

**Comment by darrenleeweber** [comment]
#295 seems to help with the logic in the `is_utc()` method, but that doesn't entirely resolve some concerns about the `pendulum.parse()` method and how it should behave, esp. when given an explicit timezone to apply.

**Comment by joaonc** [comment]
My take is that the scope of this issue is the `is_utc()` method which, regardless of how the DateTime object was created, if the timzone has an offset of 0 (no offset, aka UTC), then `is_utc()` should return `True`, which hopefully now it does.
